cf-deploy-teardown/tasks/k8s-ready-state-check.sh

```diff
@@ -1,72 +1,126 @@
 #!/bin/bash
 
+function usage() {
+    cat <<EOF
+Usage: $(basename "${0}") ?options? ?category?
```

Comment:
😀  I'm not that familiar with tcl documentation styles, but I guess this is optional parameters?